flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/nodes/physical/common/CommonPhysicalLookupJoin.scala

```diff
@@ -209,6 +215,20 @@ abstract class CommonPhysicalLookupJoin(
       .itemIf("retry", retryOptions.getOrElse(""), retryOptions.isDefined)
   }
 
+  private def getTableFilterString(t: TableSourceTable): String = {
+    val filterOpt = t.abilitySpecs.collectFirst { case spec: FilterPushDownSpec => spec }
```

Comment:
I'm wondering should `PartitionPushDownSpec` also need to be added into this part.

Reply:
Added 👍